nomenclature/definition.py

```diff
@@ -44,23 +44,18 @@ def __init__(self, path, dimensions=None):
         if not path.is_dir():
             raise NotADirectoryError(f"Definitions directory not found: {path}")
 
-        if (path / "config.yaml").exists():
-            self.config = DataStructureConfig.from_file(
-                path=path,
-                file="config.yaml",
-            )
+        if (file := path.parent / ".nomenclature.yaml").exists():

```

Comment:
I like the idea of the file name, but I worry that Windows & Mac will hide the file it it starts with `.`... And in contrast to RTD-config, this is quite important for our users.
```suggestion
        if (file := path.parent / "nomenclature.yaml").exists():

```

Reply:
Fair point, I'll remove the leading dot.